airflow/cli/commands/remote_commands/connection_command.py

```diff
@@ -64,35 +70,31 @@ def _connection_mapper(conn: Connection) -> dict[str, Any]:
 
 @suppress_logs_and_warning
 @providers_configuration_loaded
-def connections_get(args):
+@provide_cli_api_client
+def connections_get(args, cli_api_client=NEW_CLI_API_CLIENT):
     """Get a connection."""
-    try:
-        conn = BaseHook.get_connection(args.conn_id)
-    except AirflowNotFoundException:
-        raise SystemExit("Connection not found.")
```

Comment:
I think something like this is the error handling pattern we should keep

Reply:
Added them back while throwing from inner methods.